constellation_devicemanager/models.py

```diff
@@ -1,8 +1,18 @@
 from django.db import models
 from django.contrib.auth.models import User
+from django.core.validators import RegexValidator
 
 class Device(models.Model):
-    MAC = models.CharField(primary_key=True, max_length=17)
+    MAC = models.CharField(
+        primary_key=True,
+        max_length=17,
+        validators=[
+            RegexValidator(
+                regex='^([0-9A-Fa-f]{2}[:-]?){5}([0-9A-Fa-f]{2})$',
+                message='Not a MAC address',
```

Comment:
This message should include a remark that it is not a properly formatted MAC address as it might still be a valid, improperly formatted one.